integrations/gitlab.js

```diff
@@ -8,49 +8,47 @@ const { gitIssues } = require('../utils/api/gitlab')
  */
 async function listIssues(authData) {
   const { gitlabToken, projectId } = authData
-  sdk.log('')
-  ux.spinner.start('Retrieving GitLab issues')
+  await ux.print('')
```

Comment:
Is this just to put in a newline character? Can we add this to the previous print, or to the spinner start on the next line before the text?

Reply:
Actually I don't think it can be added to the spinner because it'll keep adding new lines on every re-print of the spinner

Reply:
If you add it inside the spinner it will screw up with the formatting, cause the spinner keeps on logging out and returns new lines. There is also no more afterMessage support for the prompts so I just left it with how Ethan did it

Reply:
I think you should be able to ditch all the `ux.print('')` calls I think they'll look weird on slack, you can put the newlines on the spinner.stop's or at the end of a previous ux.print's safely I think

Reply:
took them all out, formatting still looks ok